### PR TITLE
fix cloudwatch metric used to monitor for api 5xx errors

### PR DESCRIPTION
**Pull request by asjohnston-asf**

### CODE REVIEW

File: apps/api/api-cf.yml.j2 [hunk 1]
```diff
@@ -44,8 +44,8 @@ Outputs:
   Url:
     Value: {{ '!Sub "https://${RestApi}.execute-api.${AWS::Region}.amazonaws.com/${Stage}/ui"' if security_environment == 'EDC' else '!Sub "https://${CustomDomainName}/ui"' }}
 
-  ApiId:
-    Value: !Ref RestApi
+  ApiName:
+    Value: !GetAtt RestApi.RestApiId
```
Comment (jtherrmann) [inline apps/api/api-cf.yml.j2:48]: Just confirming that this should be `RestApiId`, not `RestApiName`?

**Review by jtherrmann — APPROVED** [review]
LGTM, as long as that field name is correct (see my comment above).

File: apps/api/api-cf.yml.j2 (hunk 1 above)
Comment (asjohnston-asf) [reply apps/api/api-cf.yml.j2:48]: The available return values are documented at https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-restapi.html#aws-resource-apigateway-restapi-return-values

We want the auto-generated `Imported on 2022-03-15T20:13:40Z` name. I'm not 100% confident that's `RestApiId`, but it's the obvious one.

Comment (asjohnston-asf) [reply apps/api/api-cf.yml.j2:48]: :thinking: `!Ref RestApi` and `!GetAtt RestApi.RestApi.Id` ended up returning the same value. Deployment succeeded but the alarm still won't work. Needs more research and a followup PR.

**Pull request merged**